src/js/components/stock-movement/inbound/StockMovementInboundList.jsx

```diff
@@ -22,6 +23,10 @@ const StockMovementInboundList = (props) => {
 
   useTranslation('stockMovement', 'reactTable');
 
+  useEffect(() => {
+    triggerDeletionNotification(props.location?.search);
+  }, []);
+
```

Comment:
I know we were discussing it yesterday, but now when I see the code, I see a possibility to move it to a custom hook, called maybe: `useDeleteNotificationListener`

Reply:
I am more willing to do it as a hook than keeping the logic connected with `useEffect` here.